config/routes.rb

```diff
@@ -2,7 +2,7 @@
 
   root "categories#index"
   post "/login", to: "sessions#create"
-  delete "/logout", to: "sessions#destroy", as: "logout"
+  get  "/logout", to: "sessions#destroy", as: "logout"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.